mycity/mycity/intents/sailing_weather_intent.py

```diff
@@ -0,0 +1,75 @@
+"""
+Alexa intent used to find the sailing weather conditions 
+on the Charles River
+"""
+
+import requests
+
+from mycity.mycity_response_data_model import MyCityResponseDataModel
+
+
+COMMUNITY_SAILING = "https://api2.community-boating.org/api/flag"
+
+
+def get_sailing_weather_intent(mycity_request):
+    """
+    Retrieves the sailing conditions from Community Boating's API
+    and provides it to the user in an easily digestible format.
+    
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseDataModel object
+    """
+    print(
+        '[method: get_sailing_weather_intent]',
+        'MyCityRequestDataModel received:',
+        str(mycity_request)
+    )
+    mycity_response = MyCityResponseDataModel()
+    
+    r = requests.get("https://api2.community-boating.org/api/flag")
+    
+    mycity_response.output_speech = get_sailing_conditions(r.text)
+    mycity_response.session_attributes = mycity_request.session_attributes
+    mycity_response.card_title = mycity_request.intent_name
```

Comment:
Let's make this a little more user friendly. Maybes something like "Community Boating Sailing Conditions"

Reply:
Hey Jamie, are you referring to the name of the intent, or the name of the method here?

Reply:
Whatever we set `mycity_response.card_title`. This is what is displayed on the users device if their Alexa has a screen